Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -132,21 +157,59 @@ export type NotificationProps = {
   onHide?: (id: string) => void;
 };
 
+const makeLinkLabel = (link: NotificationLink) => (
+  <ActionButton styles={linkButton} onClick={link.onClick}>
+    {link.label}
+  </ActionButton>
+);
+
 const defaultCardContentRenderer = (props: CardProps) => {
-  const { title, description, type, link } = props;
+  const { title, description, type, link, links, leftLinks, rightLinks } = props;
+
+  const rightLinkList = rightLinks ?? links ?? [link];
+  const leftLinkList = leftLinks ?? [];
+
+  const stackProps: IStackProps = {
+    horizontal: true,
+    horizontalAlign: 'space-between',
+    tokens: {
+      childrenGap: '20px',
+      padding: '0 16px 0 0',
+      maxHeight: '24px',
+    },
+  };
+
   return (
     <div css={cardContent}>
       {type === 'error' && <Icon css={errorType} iconName="ErrorBadge" />}
       {type === 'success' && <Icon css={successType} iconName="Completed" />}
       {type === 'warning' && <Icon css={warningType} iconName="Warning" />}
+      {type === 'question' && <Icon css={questionType} iconName="UnknownSolid" />}
+      {type === 'congratulation' && <Icon css={congratulationType} iconName="Trophy2Solid" />}
```

Comment:
A note here: this isn't used anywhere yet, so I'm all right leaving it out, but it makes a good indicator type for messages like "Congratulations, you've published your first bot to Azure!" or such.